graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/IndexModelBuilder.java

```diff
@@ -59,6 +59,10 @@
   IndexModel build() {
     Optional<String> indexClass =
         DirectiveHelper.getStringArgument(cqlIndexDirective, "class", context);
+    // Some persistence backends default to SAI when no class is specified:
+    if (!indexClass.isPresent() && !context.defaultsToRegularIndexes()) {
+      indexClass = Optional.of("org.apache.cassandra.index.sai.StorageAttachedIndex");
```

Comment:
Should we also check `Persistence.supportsSAI)` in this case?

Although, in practice we do not have `Persistence` implementations that support neither 2i nor SAI, we could probably produce a nicer error message if it ever happens in the future. WDYT?

Reply:
👍 you're right, I'll make the change.

Reply:
Also, maybe log the info, i.e:
`log.info("The index is not set, defaulting to sai")`
?

Reply:
👍 I added a processing message

Reply:
Maybe add a warning to the context, similar to `invalidMapping(...)` but a warning if we default to SAI? Would it be helpful to the end user?

Reply:
Yeah that's pretty much what `info()` does. The message will be visible to the user if they select `logs` in the `deploySchema` response.